common/datasets/librispeech/cart.py

```diff
@@ -0,0 +1,939 @@
+                                "values": "AA0 AA1 AA2",
+                            },
+                            {
+                                "type": "question",
+                                "description": "vowel_AE",
+                                "values": "AE0 AE1 AE2",
+                            },
+                            {
+                                "type": "question",
+                                "description": "vowel_AH",
+                                "values": "AH0 AH1 AH2",
+                            },
+                            {
+                                "type": "question",
+                                "description": "vowel_AO",
+                                "values": "AO0 AO1 AO2",
+                            },
+                            {
+                                "type": "question",
+                                "description": "vowel_IY",
+                                "values": "IY0 IY1 IY2",
+                            },
+                            {
+                                "type": "question",
+                                "description": "sz",
+                                "values": "S Z",
+                            },
+                            {
+                                "type": "question",
+                                "description": "kg",
+                                "values": "K G",
+                            },
+                            {
+                                "type": "question",
+                                "description": "vowel_EH",
+                                "values": "EH0 EH1 EH2",
+                            },
+                            {
+                                "type": "question",
+                                "description": "vowel_OY",
+                                "values": "OY0 OY1 OY2",
+                            },
+                            {
+                                "type": "question",
+                                "description": "vowel_OW",
+                                "values": "OW0 OW1 OW2",
+                            },
+                            {
+                                "type": "question",
+                                "description": "vowel_IH",
+                                "values": "IH0 IH1 IH2",
+                            },
+                            {
+                                "type": "question",
+                                "description": "vowel_EY",
+                                "values": "EY0 EY1 EY2",
+                            },
+                            {
+                                "type": "question",
+                                "description": "vowel_ER",
+                                "values": "ER0 ER1 ER2",
+                            },
+                        ],
+                    },
+                ],
+            },
+        ]
```

Comment:
here the logic to remove the unknown question is missing. request changes also for this one.